langchain/vectorstores/deeplake.py

```diff
@@ -171,11 +172,19 @@ def add_texts(
             texts (Iterable[str]): Texts to add to the vectorstore.
             metadatas (Optional[List[dict]], optional): Optional list of metadatas.
             ids (Optional[List[str]], optional): Optional list of IDs.
-            **kwargs: other optional keyword arguments.
+            embedding_function (Optional[Embeddings], optional): Embedding function
+                to use to convert the text into embeddings.
+            **kwargs (Any): Additional keyword arguments
 
         Returns:
             List[str]: List of IDs of the added texts.
         """
+        if kwargs:
+            first_item = next(iter(kwargs))
+            raise TypeError(
+                f"`{first_item}` is not a valid argument to add_text method"
```

Comment:
should we just pass in all of kwargs, otherwise users won't know that the rest of kwargs was also erroneous